### DISCUSSION TITLE
希望进展列表里包含清单进展

### DISCUSSION THREAD

**Discussion by SUNFUNoo** [WatsonYao/nian_app#127] [Q&A] [answered]
呜呜呜，进展列表里除了清单其他啥都有。希望清单记本里的内容，也可以在首页左滑的进展列表里出现🥺。
如果不能在列表里显示全部清单的话，那可不可以只显示已完成的清单呢🤔

**Comment by WatsonYao** [comment, marked as answer]
考虑到清单都是多条一起查看的，首页右屏展示单条感觉怪怪的。
这边先记下来，如果方便展示的话，后续版本会试试展示已完成的清单。

**Reply by SUNFUNoo** [reply]
好嘞好嘞感谢🤩
我觉得清单和计时、打卡这些，从某种程度上来说都是对过往时刻的记录（无论记录的内容是已经完成的事项、已经过去的心情还是其他），那么也是可以囊括在一天的时间轴里的。
把这些都展示在进展列表里，也便于以后对这一天进行复盘。

**Reply by WatsonYao** [reply]
对了，你这一提，倒是想起来了，日历里面选中某天，下面会列出当天的这些进展，包括清单的。

**Reply by SUNFUNoo** [reply]
好的找到了，不过日历里的是按照清单添加时间排序的。
但是很多时候我是稍微有点想法就会添加到清单里，真正去做去完成这个事情的时候，往往是好几天之后了🤣
所以希望也有可以按照清单完成时间来排列每天的时间轴的功能。
